compiler/compile/OMRSymbolReferenceTable.cpp

```diff
@@ -676,6 +676,19 @@ OMR::SymbolReferenceTable::findOrCreateCurrentTimeMaxPrecisionSymbol()
    return element(currentTimeMaxPrecisionSymbol);
    }
 
+TR::SymbolReference *
+OMR::SymbolReferenceTable::findOrCreateEncodeASCIISymbol()
```

Comment:
`encodeASCIISymbol` does need to be defined in OMR (unfortunately), but this method could be defined in OpenJ9 to reduce the amount of OpenJ9-specific code being added here